src/validation.cpp

```diff
@@ -3553,7 +3553,7 @@ bool Chainstate::ActivateBestChain(BlockValidationState& state, std::shared_ptr<
             } // release MempoolMutex
             // Notify external listeners about the new tip, even if pindexFork == pindexNewTip.
             if (m_chainman.m_options.signals && this == &m_chainman.ActiveChainstate()) {
-                m_chainman.m_options.signals->ActiveTipChange(pindexNewTip, m_chainman.IsInitialBlockDownload());
+                m_chainman.m_options.signals->ActiveTipChange(*pindexNewTip, m_chainman.IsInitialBlockDownload());
```

Comment:
aside: we could throw an `Assume()` on the pointer before deref, since it's not set in a non-trivial way above

Reply:
(If you're gonna do this) `Assert` would make more sense, since we'd never want to hit the deref if it fails

Reply:
I could add `Assume` to the `if` condition?